data/xml/W19.xml

```diff
@@ -13922,4 +13922,460 @@ In this tutorial on MT and post-editing we would like to continue sharing the la
+      <author><first>Vaclava</first><last>Kettnerova</last></author>
+      <author><first>Marketa</first><last>Lopatkova</last></author>
+      <pages>14–25</pages>
+      <url>W19-7704</url>
+    </paper>
+    <paper id="5">
+      <title>Coordination of Unlike Grammatical Functions</title>
+      <author><first>Agnieszka</first><last>Patejuk</last></author>
+      <author><first>Adam</first><last>Przepiórkowski</last></author>
+      <pages>26–37</pages>
+      <url>W19-7705</url>
+    </paper>
+    <paper id="6">
+      <title>Predicate Catenae: A Dependency Grammar Analysis of It-Clefts</title>
+      <author><first>Timothy</first><last>Osborne</last></author>
+      <pages>38–48</pages>
+      <url>W19-7706</url>
+    </paper>
+    <paper id="7">
+      <title>Noun Phrases Rooted by Adjectives: A Dependency Grammar Analysis of the Big Mess Construction</title>
+      <author><first>Timothy</first><last>Osborne</last></author>
+      <pages>49–59</pages>
+      <url>W19-7707</url>
+    </paper>
+    <paper id="8">
+      <title>Cliticization of <fixed-case>S</fixed-case>erbian Personal Pronouns and Auxiliary Verbs. A Dependency-Based Account</title>
+      <author><first>Jasmina</first><last>Milicevic</last></author>
+      <pages>60–68</pages>
+      <url>W19-7708</url>
+    </paper>
+    <paper id="9">
+      <title>The evolution of spatial rationales in Tesnière’s stemmas</title>
+      <author><first>Nicolas</first><last>Mazziotta</last></author>
+      <pages>69–80</pages>
+      <url>W19-7709</url>
+    </paper>
+    <paper id="10">
+      <title>Toward a cognitive dependency grammar of <fixed-case>H</fixed-case>ungarian</title>
+      <author><first>András</first><last>Imrényi</last></author>
+      <pages>81–88</pages>
+      <url>W19-7710</url>
+    </paper>
+    <paper id="11">
+      <title>Interpreting and defining connections in a dependency structure</title>
+      <author><first>Sylvain</first><last>Kahane</last></author>
+      <pages>89–99</pages>
+      <url>W19-7711</url>
+    </paper>
+    <paper id="12">
+      <title>Identifying Grammar Rules for Language Education with Dependency Parsing in <fixed-case>G</fixed-case>erman</title>
+      <author><first>Eleni</first><last>Metheniti</last></author>
+      <author><first>Pomi</first><last>Park</last></author>
+      <author><first>Kristina</first><last>Kolesova</last></author>
+      <author><first>Günter</first><last>Neumann</last></author>
+      <pages>100–111</pages>
+      <url>W19-7712</url>
+    </paper>
+    <paper id="13">
+      <title>How to Parse Low-Resource Languages: Cross-Lingual Parsing, Target Language Annotation, or Both?</title>
+      <author><first>Ailsa</first><last>Meechan-Maddon</last></author>
+      <author><first>Joakim</first><last>Nivre</last></author>
+      <pages>112–120</pages>
+      <url>W19-7713</url>
+    </paper>
+    <paper id="14">
+      <title>Word order variation in Mbyá Guaraní</title>
```

Comment:
The language name should be fixed-case.

Reply:
Can you pull down this branch and add these missed entries to `bin/fixedcase/truelist`?

Reply:
Actually I did it